lib/slack_bot_server/bot.rb

```diff
@@ -12,49 +11,46 @@ class InvalidToken < RuntimeError; end
   def initialize(token:, key: nil)
     @token = token
     @key = key || @token
-    @api = ::Slack::Client.new(token: @token)
-    @im_channel_ids = []
-    @channel_ids = []
+    @client = ::Slack::RealTime::Client.new(token: @token)
+    @im_channels = []
+    @channels = []
     @connected = false
     @running = false
-    @next_message_id = 0
 
-    raise InvalidToken unless rtm_start_data['ok']
+    raise InvalidToken unless @client.web_client.auth_test['ok']
```

Comment:
You shouldn't need this anymore unless you're injecting a custom Faraday connection, the library will take care of raising a `Slack::Web::Api::Error` if there's anything other than an `'ok` in the body [here](https://github.com/dblock/slack-ruby-client/blob/636f6c1d9b2f926b64a91a16190ca09fdc19340e/lib/slack/web/faraday/response/raise_error.rb#L8). If you want to handle and re-raise `InvalidToken` you can too, but you're also combining `account_inactive` and `invalid_auth`, and you probably want to just bubble up the original exception to deal with clients that have been disabled.